lib/connection.js

```diff
@@ -33,6 +33,8 @@ function Connection(options = {}) {
   let next = false
   let statements = {}
   let connect_timer
+  let bs = null
```

Comment:
Maybe naming this as `bufferStack` or `bufferQueue` would help to understand what's going on 6 months later :D

Reply:
Good idea 😉 But `bs` is such a nice variable name 🤪